tests/test_glm_synthetic.py

```diff
@@ -16,7 +16,7 @@
     n_basis_funcs=5,
     window_size=ws
 )
-sim_pts = nsl.sample_points.raised_cosine_log(5, ws)
+sim_pts = jnp.clip(neurostatslib.sample_points.raised_cosine_log(5, ws),0,100)
```

Comment:
this shouldn't be necessary with the change in `raised_cosine_log`, right?

Reply:
now yes, anyways, this code which is more for visualization will be moved to a dedicated documentation script.